jdk 11 ubuntu build failed with 'Exception in thread "main" java.lang.RuntimeException: Failed to load alias map from resource: /com/ibm/j9ddr/StructureAliases29.dat - cannot continue'

jdk 11 ubuntu build failed with message: 'Exception in thread "main" java.lang.RuntimeException: Failed to load alias map from resource: /com/ibm/j9ddr/StructureAliases29.dat - cannot continue'  from May 12 or May 13th.

```
Generating DDR pointer and structure class files
Exception in thread "main" java.lang.RuntimeException: Failed to load alias map from resource: /com/ibm/j9ddr/StructureAliases29.dat - cannot continue
	at com.ibm.j9ddr.StructureReader.loadAliasMapData(StructureReader.java:438)
	at com.ibm.j9ddr.StructureReader.loadAliasMap(StructureReader.java:306)
	at com.ibm.j9ddr.StructureReader.applyAliases(StructureReader.java:222)
	at com.ibm.j9ddr.StructureReader.<init>(StructureReader.java:120)
	at com.ibm.j9ddr.tools.ClassGenerator.readBlob(ClassGenerator.java:93)
	at com.ibm.j9ddr.tools.ClassGenerator.run(ClassGenerator.java:174)
	at com.ibm.j9ddr.tools.ClassGenerator.main(ClassGenerator.java:79)
make[4]: *** [/home/runner/work/build-openj9/build-openj9/openj9-openjdk-jdk11/build/linux-x86_64-normal-server-release/support/ddr/classes.done] Error 1
/home/runner/work/build-openj9/build-openj9/openj9-openjdk-jdk11/closed/DDR.gmk:106: recipe for target '/home/runner/work/build-openj9/build-openj9/openj9-openjdk-jdk11/build/linux-x86_64-normal-server-release/support/ddr/classes.done' failed
Gensrc-openj9.dtfj.gmk:25: recipe for target 'all' failed
make[3]: *** [all] Error 2
make/Main.gmk:112: recipe for target 'openj9.dtfj-gensrc-src' failed
make[2]: *** [openj9.dtfj-gensrc-src] Error 1
```
https://github.com/eclipse-openj9/build-openj9/runs/2659808852?check_suite_focus=true

--- a/src/builder.ts
+++ b/src/builder.ts
@@ -217,8 +217,12 @@ async function getBootJdk(version: string): Promise<void> {
   if (parseInt(bootJDKVersion) > 8) {
     let bootjdkJar
     // TODO: issue open openj9,mac, 10 ga : https://api.adoptopenjdk.net/v3/binary/latest/10/ga/mac/x64/jdk/openj9/normal/adoptopenjdk doesn't work
-    if (`${bootJDKVersion}` === '10' && `${targetOs}` === 'mac') {
-      bootjdkJar = await tc.downloadTool(`https://github.com/AdoptOpenJDK/openjdk10-binaries/releases/download/jdk-10.0.2%2B13.1/OpenJDK10U-jdk_x64_mac_hotspot_10.0.2_13.tar.gz`)
+    if (`${bootJDKVersion}` === '10') {
+      //JDK 11 require a latest jdk11 as boot JVMhttps://github.com/eclipse-openj9/build-openj9/issues/25#issuecomment-848354452
+      bootjdkJar = await tc.downloadTool(`https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk11u-2021-05-07-07-34/OpenJDK11U-jdk_x64_linux_openj9_2021-05-07-07-34.tar.gz`)
+      if (`${targetOs}` === 'mac') {
+        bootjdkJar = await tc.downloadTool(`https://github.com/AdoptOpenJDK/openjdk10-binaries/releases/download/jdk-10.0.2%2B13.1/OpenJDK10U-jdk_x64_mac_hotspot_10.0.2_13.tar.gz`)
+      }
     } else {
       bootjdkJar = await tc.downloadTool(`https://api.adoptopenjdk.net/v3/binary/latest/${bootJDKVersion}/ga/${targetOs}/x64/jdk/openj9/normal/adoptopenjdk`)
     }
@@ -226,7 +230,7 @@ async function getBootJdk(version: string): Promise<void> {
     if (`${targetOs}` === 'mac') {
       await exec.exec(`sudo tar -xzf ${bootjdkJar} -C ./bootjdk --strip=3`)
     } else if (`${targetOs}` === 'linux') {
-      if (`${bootJDKVersion}` === '10' ) {
+      if (`${bootJDKVersion}` === '10' && `${targetOs}` === 'mac' ) {
         await exec.exec(`sudo tar -xzf ${bootjdkJar} -C ./bootjdk --strip=2`) // TODO : issue open as this is packaged differently
       } else {
         await exec.exec(`sudo tar -xzf ${bootjdkJar} -C ./bootjdk --strip=1`)

